opts/secret.go

```diff
@@ -50,7 +50,7 @@ func (o *SecretOpt) Set(value string) error {
 
 		value := parts[1]
 		switch key {
-		case "source":
+		case "source", "src":
 			options.Source = value
 		case "target":
```

Comment:
Correct.  We chose to stay away from "destination" because the `target` is forcibly created under `/run` at a single level (i.e. `/run/secrets/foo`).